### DISCUSSION TITLE
Skiline in Mali G52

### DISCUSSION THREAD

**Discussion by SansGamez** [skyline-emu/skyline#1874] [Ideas]
Plis

**Comment by parknich082** [comment]
stop crying about you alls cheap ass phones not working emulating a fucking nintendo switch, the devs will fix it when they fix it. this emulator is still new, theres more important stuff than making your alls shitty phone work. get over it. if you want it fixed so bad, then fix it yourself. its open source.

**Reply by Fanatic4** [reply]
And I am here crying in iOS 😂😂😂
